HARK/cAndCwithStickyE/StickyEtools.py

```diff
@@ -185,7 +189,7 @@ def makeStickyEdataFile(Economy,ignore_periods,description='',filename=None,save
                 DataArray = np.hstack((DataArray,np.reshape(R,(R.size,1))))
                 VarNames.append('R')
             with open(results_dir + filename + 'Data.txt','wb') as f:
-                my_writer = csv.writer(f, delimiter = '\t')
+                my_writer = csv.writer(f, delimiter = r'\t')
```

Comment:
In particular, I don't think this does what you want it to do - it looks like it will write a CSV file with a delimiter of the character `\` followed by the character `t`, rather than a tab character.

Reply:
Yes, this has been fixed.